crashtracker-ffi/src/crash_info/sig_info.rs

```diff
@@ -2,22 +2,22 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use datadog_crashtracker::{SiCodes, SignalNames};
-use ddcommon_ffi::{slice::AsBytes, CharSlice};
+use ddcommon_ffi::ToHexStr;
 
 #[repr(C)]
-pub struct SigInfo<'a> {
-    pub addr: CharSlice<'a>,
+pub struct SigInfo {
+    pub addr: usize,
```

Comment:
Are we guaranteed to always have one of these?

Reply:
good point

Reply:
rolled back to the previous code